COHHIO/ryha
# Append CoC code to each project in filter

# Overview
Main development:
- CoC code was added next to project name choices in "Project" dropdown. Several changes were necessary to make this change (check Details for more information). These changes are meant to be incorporated once the database is reset.

In this PR I also included some minor changes:
- cleaned database-utils scripts (some code styling, file renaming and location changes)
- minor documentation update (there was an argument that wasn't used and was present in examples)

# Details
CoC code is present in ProjectCoC.csv, a file that is not currently processed in the app. I used Enrollment.csv processing as a reference, because the files have some characteristics in common (the need to use project_id instead of orig_project_id being the most noteworthy).

Required changes included:
- create read function to read .csv data (`read_project_coc()`)
- add SQL script that creates project_coc table in the database
- modify HMISmetadata to reflect that a new file is required
- include project_coc logic when creating dm object
- properly append coc code to project name
- make dropdown-options container be the body to solve a UI issue

# Test
- Run postgres/utils/recreate_dev.R
- Launch the application in dev and upload a test file
- Relaunch the application (to update the dev database)
- Check that coc code was appended accordingly

Closes #47
